gradle/scripts/BranchName.gradle

```diff
@@ -0,0 +1,23 @@
+task checkBranchName {
+  doLast {
+    def branchName = System.getenv('GITHUB_REF_NAME')
+    def eventName = System.getenv('GITHUB_EVENT_NAME')
+    def headRef = System.getenv('GITHUB_HEAD_REF')
```

Comment:
Would be great if you could pass the branch name variable to the task like I mentioned in [this comment](https://github.com/ie3-institute/simona/pull/1011#issuecomment-2483029978). (I did not point to the right resource though, it should be [this](https://docs.gradle.org/current/userguide/project_properties.html))

So, it'd be great to find the proper branch name within ci.yml, and then call the task somehow like this:
```bash
./gradlew -PbranchName=[...] checkBranchName
```

So this way, we would handle GitHub-specific stuff within the file related to Github, and the rest within gradle. It works with environment variables too, though, if you don't like the `-P` option.

Reply:
Should work like that now ;)